kuma/wiki/middleware.py

```diff
@@ -51,6 +51,17 @@ def process_request(self, request):
 
                 return HttpResponseRedirect(new_path)
 
+            elif request.path_info == u'/docs{}'.format(new_path):
+                # Is this a request for a DocumentZone, but with /docs/ wedged
+                # in the url path between the language code and the zone's url_root?
+                new_path = u'/{}{}'.format(request.LANGUAGE_CODE, new_path)
+                query = request.GET.copy()
+                if 'lang' in query:
+                    query.pop('lang')
```

Comment:
Ok, will do.  That sounds small enough, I can probably do it during my lunch break.